htlcswitch/link_test.go

```diff
@@ -1946,6 +1946,7 @@ func newSingleLinkTestHarness(chanAmt, chanReserve btcutil.Amount) (
 	cleanUp := func() {
 		close(alicePeer.quit)
 		defer fCleanUp()
+		defer invoiceRegistry.cleanup()
```

Comment:
Can remove the `defer` for both of these calls as they'd be triggered immediately anyway.

Reply:
Fixed!